### PR TITLE
add acknowledgments section

### LINKED ISSUE #23: Needs acknowledgments section
See e.g. https://fetch.spec.whatwg.org/#acknowledgments or https://streams.spec.whatwg.org/#acks

This is actually pretty important since that is where the CC0 declaration goes.

### PR DESCRIPTION
**Pull request by robertkowalski**
closes #23

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -287,3 +287,18 @@ Then, perform Logger("info", «<var>label</var>, <var>duration</var>»).
...
 <h2 id="acks" class="no-num">Acknowledgments</h2>
+
+The editors would like to thank
+Corey Farwell,
+Jeff Carpenter,
+Justin Woo,
+Paul Irish
+for their contributions to this specification. You are awesome!
+
+This standard is written by <a href="http://terinstock.com">Terin Stock</a> (<a
```
Comment (domenic) [inline index.bs]: https for Terin's URL

File: index.bs [hunk 2]
```diff
@@ -287,3 +287,18 @@ Then, perform Logger("info", «<var>label</var>, <var>duration</var>»).
 TODO: Define an interface that allows JavaScript objects to format themselves for inspection.
 
 <h2 id="acks" class="no-num">Acknowledgments</h2>
+
+The editors would like to thank
+Corey Farwell,
+Jeff Carpenter,
+Justin Woo,
+Paul Irish
```
Comment (domenic) [inline index.bs]: "and" before last name

**Comment by robertkowalski** [comment]
@domenic fixed

**Comment by terinjokes** [comment]
LGTM :+1:

**Comment by domenic** [comment]
LGTM, thanks for the mention! <3

**Comment by robertkowalski** [comment]
merged as 7ba833ec4469c20b52e9b58e91bf00c4af0fca86 :)

**Pull request closed without merging**